### PR TITLE
[Remove Stream/Optional usage] LPS-176500 module: dynamic-data-mapping-form-field-type Part2

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-176500

previous:  https://github.com/liferay-core-infra/liferay-portal/pull/3621

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:data-engine

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/localizable/text/LocalizableTextDDMFormFieldTemplateContextContributor.java [hunk 1]
```diff
@@ -171,15 +158,10 @@ private JSONArray _getPlaceholdersSubmitLabelJSONArray() {
...
-		Set<Locale> availableLocales = _language.getAvailableLocales();
-
-		Stream<Locale> stream = availableLocales.stream();
-
-		stream.map(
-			this::_getPlaceholdersSubmitLabelJSONObject
-		).forEach(
-			placeholdersSubmitLabelJSONArray::put
-		);
+		for (Locale locale : _language.getAvailableLocales()) {
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/localizable/text/LocalizableTextDDMFormFieldTemplateContextContributor.java:161]: @JanisZhang Please also apply JSONUtil.toJSONArray to this one.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/redirect/button/RedirectButtonDDMFormFieldTemplateContextContributor.java [hunk 2]
```diff
@@ -84,32 +83,34 @@ public Map<String, Object> getParameters(
 					RequestBackedPortletURLFactoryUtil.create(
 						ddmFormFieldRenderingContext.getHttpServletRequest());
 
+				Map<String, String[]> parameterMap = new HashMap<>();
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/redirect/button/RedirectButtonDDMFormFieldTemplateContextContributor.java:86]: @JanisZhang Please rename the array in line 93 to `parameterPair` and rename this to `parameters`.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**